### DISCUSSION TITLE
Adding supports

### DISCUSSION THREAD

**Discussion by Sumnium111** [OrcaSlicer/OrcaSlicer#1735] [Q&A]
Hello, I am new to 3D printing. I have a question how to add supports myself?

**Comment by Glenn1963** [comment]
![image](https://github.com/SoftFever/OrcaSlicer/assets/17497231/b9374561-ec42-434a-8d05-2fed96509e30)
If you select an object, one of the enabled menu items is Support Painting. It provides a few options for you to add supports where you want.
You will also want to change some Support settings to enable Support and then choose if you want only your manual support or if you want to add your painted support to the Automatic generation
![image](https://github.com/SoftFever/OrcaSlicer/assets/17497231/e9820f1f-ddde-43b3-a58f-cc91a81561ad)

**Reply by Sumnium111** [reply]
thank you very much for your help
